circleci/resource_circleci_environment_variable.go

```diff
@@ -168,3 +189,29 @@ func getOrganization(d *schema.ResourceData, providerClient *ProviderClient) *st
 
 	return providerClient.organization
 }
+
+func getEnvironmentVariableId(d *schema.ResourceData) error {
+	parts := parseEnvironmentVariableId(d.Id())
+	// Validate that he have values for all the ID segments. This should be at least 3
+	if parts[0] == "" || parts[1] == "" || parts[2] == "" {
+		return fmt.Errorf("error calculating circle_ci_environment_variable. Please make sure the ID is in the form ORGANIZATION.PROJECTNAME.VARNAME (i.e. foo.bar.my_var)")
+	}
+
+	d.Set("organization", parts[0])
+	d.Set("project", parts[1])
+	d.Set("name", parts[2])
+	return nil
+}
+
+func parseEnvironmentVariableId(id string) [3]string {
```

Comment:
Why not make this function return 3 strings `(organization, projectName, envName string)` instead of wrapping them in a array ?

Reply:
I didn't even think of that. I'll convert to this instead as it makes it much more readable